test/functional/apps/dashboard/exports/timezonetest_6_2_4.json

```diff
@@ -1,25 +0,0 @@
-[
-  {
-    "_id": "61d06bc0-5215-11e8-82a7-0b6f5e824cae",
-    "_type": "dashboard",
-    "_source": {
-      "title": "time zone test",
-      "hits": 0,
-      "description": "",
-      "panelsJSON": "[{\"gridData\":{\"h\":3,\"i\":\"1\",\"w\":6,\"x\":0,\"y\":0},\"id\":\"3fe22200-3dcb-11e8-8660-4d65aa086b3c\",\"panelIndex\":\"1\",\"type\":\"visualization\",\"version\":\"6.2.4\"}]",
-      "optionsJSON": "{\"hidePanelTitles\":false,\"useMargins\":true}",
```

Comment:
Is that fine to remove those files/tests , or should we just skip the tests and open an issue for the presentation team to adapt them?

Reply:
Um, good idea!